src/react/components/pages/modelCompose/composeModelView.tsx

```diff
@@ -95,6 +95,7 @@ export default class ComposeModelView extends React.Component<IComposeModelViewP
                         <PrimaryButton
                             className="model-confirm"
                             theme={getPrimaryGreenTheme()}
+                            disabled={this.state.items.length <= 1 ? true : false}
```

Comment:
.length  should  be at least 2 since you need at least two model to compose

Reply:
Yeah, then less than and equal to the disabled will be set to true, which will disable the button

Reply:
Also just disable button seems to be odd -we need to add dialog user chooses less than to model 
like: "You need to select at least two models". And do not show compose model if they did not select 2 models

Reply:
Yeah, it is true, I will make some changes on it